aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml

```diff
@@ -14,7 +14,7 @@
 
 apiVersion: v2
 name: kubernetes-dashboard
-version: 5.4.0
+version: 5.4.1
```

Comment:
```suggestion
version: 5.5.0
```
According to semver, this is new feature.

Reply:
yes, sorry. Done